datasets/dataset_imdb.py

```diff
@@ -0,0 +1,60 @@
+# -*- coding: utf-8 -*-
+"""
+IMDB dataset.
+inspired from:
+https://keras.io/examples/nlp/bidirectional_lstm_imdb/
+"""
+
+import keras
+from keras.utils import to_categorical
+from keras.preprocessing.sequence import pad_sequences
+from keras import models, layers
+from keras.datasets import imdb
+
+out_of_vocabulary = 2
+# We keep the 10000 first more frequent words in the datasets
+# Also the dimentionality of the input of the Embedding layer
+size_vocabulary = 20000
```

Comment:
Two comments here:

1. Unclear if this is 10000 or 20000
1. Just for readibility, maybe something more like:
   ```py
   # Definition of dataset-specific parameters
   size_vocabulary = ...  # Explanation...
   maxlen = ...  # Explanation...
   ...
   ```